src/components/ui/Navbar/Navbar.mdx

```diff
@@ -14,7 +14,7 @@ import { GlobalStyles } from '../../../theme/GlobalStyles';
 
 The Navbar component displays a vertical menu with navigation icon buttons and labels. Navbar is constructed using the NavbarHeader and NavbarItem components.
 
-Navbars support tooltips on any buttons by adding a `data-tooltip` property. Adding a `data-tooltip-position` property controls the tooltip position
+Navbars support tooltips on any buttons by adding a `data-tooltip` property. Adding a `data-tooltip-position` property controls the tooltip position. It will render the `label` as the tooltip, unless a `tooltipContent` prop is passed.
```

Comment:
Navbar does not support tooltipContent right?

Reply:
Yeah that should be `NavbarItem`, not `Navbar`

Reply:
I actually mean NavbarItem does not support tooltipContent right?

Reply:
It does. We're spreading props, so we don't need to explicitly call it out